src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs

```diff
@@ -654,7 +912,12 @@ internal ImportXmlHelper(string fileName, PSCmdlet cmdlet, bool isLiteralPath)
         internal FileStream _fs;
 
         /// <summary>
-        /// XmlReader used to read file.
+        /// Handle to string builder.
+        /// </summary>
+        internal StringBuilder _StringBuilder;
```

Comment:
```suggestion
        internal StringBuilder _stringBuilder;
```
Perhaps we could use more informative name?

Reply:
I am not sure what to call it. I contains the string array and is used to create the XmlReader  for Deserializer().
Also can we change it to private this will also fix the CodeFactor issue.

Reply:
Please convert to private if possible.